screens/eventMapPage.js

```diff
@@ -10,12 +10,27 @@ class eventMapPage extends Component{
       initLocation : {
         latitude: 42.055984,
         longitude: -87.675171,
-      }
+      },
+      currentLocation: {"coords": {"latitude": 42.055984, "longitude": -87.708}}
     }
   }
 
+  componentDidMount() {
+    this.getLocation()
+  }
+
+  async getLocation(){
+      let { status } = await Location.requestForegroundPermissionsAsync();
+      if (status !== 'granted') {
+        console.log('Permission to access location was denied');
+        return;
+      }
+
+      let location = await Location.getCurrentPositionAsync({});
+      this.setState({currentLocation: location});
+  };
+
   render() {
```

Comment:
nit: I was thinking about keeping the comment for future development work. Currently, we are hardcoding the marker coordinates but in the future we are definitely using real data to show different events. It might be a good idea to keep the comment haha